savoirfairelinux/sous-chef
# Continue dockerize the application.

## Fixes #439 

### Changes proposed in this pull request:

* Add a Nginx docker container.
* Add gunicorn as WSGI server.
* Add differents docker-compose configurations files for DEV and PROD env.
* Add a volume to made database files persistent so it could backup it later.
* Update INSTALL.md file to explain how to access to the application using the
Nginx server and how to run the Django development server.
* Remove some annoying "print()" in the tests.
* Update the .po files.


### Status


### How to verify this change

* Stop all your containers (ctrl+c) in the terminal where docker-compose run.
* Remove the containers (`docker-compose rm`)
* Remove the images (`docker rmi image_name`) you can find the names with `docker images` 
* Build all again (`docker-compose build`)
* Run all again with the development docker-compose settings (`docker-compose up`) or with the production docker-compose settings (`docker-compose -f docker-compose.yml -f docker-compose-prod.yml up`)
* Verify that the application is accessible at http://127.0.0.1 (If your port 80 is already in use stop the application that uses it before)
* Read the INSTALL.md to learn how to run the Django development server.
* You also need to run all the application migrations and add the superuser again. 

### New translatable strings
